website/docs/docs/contributing/documenting-a-new-adapter.md

```diff
@@ -0,0 +1,56 @@
+---
+title: "Documenting a new adapter"
+id: "documenting-a-new-adapter"
+---
+
+If you've already [built an adapter](/website/docs/docs/contributing/building-a-new-adapter.md), and [tested your adapter]((/website/docs/docs/contributing/testing-a-new-adapter.md)), it's time to document it so the dbt community will know that it exists and how to use it!
+
+## General Guidelines
+To best inform the dbt community of the new adapter, you will be contribtute to dbt's open-source documentation site, which uses the [Docusauraus project](https://docusaurus.io/).
+
+### Conventions
+
+Each `.md` file you create needs a header that looks like the below. The document id given below will also be need to be added to the config file: `website/sidebars.js`.
+
+```
+---
+title: "Documenting a new adapter"
+id: "documenting-a-new-adapter"
+---
+```
+
+### Single Source of Truth
+
+We asked our adapter maintainers to use the [docs.getdbt.com repo](https://github.com/dbt-labs/docs.getdbt.com) (i.e. this site) as the single-source-of-truth for documentation rather than having to maintain the same set of information in three different places. The adapter repo's `README.md` and the data platform's documentation pages should simply link to the corresponding page on this docs site.
```

Comment:
I feel like we need to offer more guidance on what belongs on the data platform's docs site vs ours.  Like installation and dbt configurations should go on our site but like how the adapter works with any of their product features might make sure on theirs (or something like that)

Reply:
i added more info. the sections below should address your concern now?